utils/kill.js

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env node
+const kill = require('kill-port');
+
+const apiPorts = [3003, 3002, 3503];
+const docHandlerPorts = [3001, 3501];
+const messageProxyPorts = [3000, 3500];
+
+const ports = [...apiPorts, ...docHandlerPorts, ...messageProxyPorts];
+
+const run = async () => {
+    return Promise.all(ports.map((port) => kill(port).catch((err) => console.log(`port ${port}:`, err.message))));
+};
+
+run();
```

Comment:
`yarn dev` is not stoped by `ctrl-c` for you too?

Reply:
yeap
it's a temp solution until a better `dev environment` setup will be implemented